backend/tracim_backend/models/tracim_session.py

```diff
@@ -12,13 +16,34 @@ class TracimSession(Session):
     def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
         self._allow_revision_deletion = False  # type: bool
+        self._context = None  # type: typing.Optional[weakref.CallableProxyType]
+
+    # TODO S.G 2020-06-08: this is a temporary setup until #1834 is done
```

Comment:
That's effectively not really nice, reason to do this is for getting the proper plugin_manager ?

Reply:
Yes, to be able to do the verification in XxxApi classes.

Reply:
Since #1854 will likely lead to give `TracimContext` (or something similar) to XxxApi instead of session/current user/config/..., I think it is acceptable.